### PR TITLE
Tilt to damage

### PR DESCRIPTION
**Pull request by namebee**
Please test before accepting.

### CODE REVIEW

File: src/main/java/io/github/protonmc/proton/mixin/LivingEntityMixin.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(LivingEntity.class)
+public class LivingEntityMixin {
+
+    @Inject(at = @At("HEAD"), method = "takeKnockback")
+    @FromModule(TiltToDamage.class)
+    public void takeKnockback(float f, double d, double e, CallbackInfo ci) {
```
Comment (YTG1234) [inline src/main/java/io/github/protonmc/proton/mixin/LivingEntityMixin.java:19]: Very descriptive parameter names 👍

File: src/main/java/io/github/protonmc/proton/mixin/LivingEntityMixin.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package io.github.protonmc.proton.mixin;
```
Comment (YTG1234) [inline src/main/java/io/github/protonmc/proton/mixin/LivingEntityMixin.java:1]: should be inside a `tweaks` sub-package or a `tweaks.tilt_to_damage`

Comment (YTG1234) [inline src/main/java/io/github/protonmc/proton/mixin/LivingEntityMixin.java]: should be inside a `tweaks` sub-package or a `tweaks.tilt_to_damage` sub-package

File: src/main/java/io/github/protonmc/proton/module/tweaks/TiltToDamage.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import net.fabricmc.fabric.api.network.ServerSidePacketRegistry;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.network.PacketByteBuf;
+import net.minecraft.util.Identifier;
+
+public class TiltToDamage extends ProtonModule {
+    public static final Identifier TILT_TO_DAMAGE_PACKET_ID = Proton.identifier("tilt_to_damage_packet");
+
+    public TiltToDamage() {
+        super(Proton.identifier("tilt_to_damage"));
```
Comment (YTG1234) [inline src/main/java/io/github/protonmc/proton/module/tweaks/TiltToDamage.java:17]: Missing lang entries

**Review by YTG1234 — CHANGES_REQUESTED** [review]

**Comment by YTG1234** [comment]
functionality works, just fix my requested changes and I'll approve

**Review by YTG1234 — APPROVED** [review]
Yes

**Review by hYdos — APPROVED** [review]

**Comment by YTG1234** [comment]
I think this is ready to merge

**Pull request merged**